src/main/kotlin/frc/robot/subsystems/Visualizer.kt

```diff
@@ -10,88 +9,225 @@ import frc.robot.lib.getRotation3d
 import frc.robot.lib.getTranslation3d
 import org.littletonrobotics.junction.AutoLogOutput
 
+private val INITIAL_MODULE_0_Turn_TRANSLATION =
+    getTranslation3d(
+        x = Meters.of(0.25448),
+        y = Meters.of(0.25448),
+        z = Meters.of(0.0508)
+    )
+private val INITIAL_MODULE_0_Drive_TRANSLATION =
+    getTranslation3d(
+        x = Meters.of(0.25448),
+        y = Meters.of(0.25448),
+        z = Meters.of(0.0508)
+    )
+
+private val INITIAL_MODULE_1_Turn_TRANSLATION =
+    getTranslation3d(
+        x = Meters.of(0.25448),
+        y = Meters.of(-0.25448),
+        z = Meters.of(0.0508)
+    )
+private val INITIAL_MODULE_1_Drive_TRANSLATION =
+    getTranslation3d(
+        x = Meters.of(0.25448),
+        y = Meters.of(-0.25448),
+        z = Meters.of(0.0508)
+    )
+
+private val INITIAL_MODULE_2_Turn_TRANSLATION =
+    getTranslation3d(
+        x = Meters.of(-0.25448),
+        y = Meters.of(0.25448),
+        z = Meters.of(0.0508)
+    )
+private val INITIAL_MODULE_2_Drive_TRANSLATION =
+    getTranslation3d(
+        x = Meters.of(-0.25448),
+        y = Meters.of(0.25448),
+        z = Meters.of(0.0508)
+    )
+
+private val INITIAL_MODULE_3_Turn_TRANSLATION =
+    getTranslation3d(
+        x = Meters.of(-0.25448),
+        y = Meters.of(-0.25448),
+        z = Meters.of(0.0508)
+    )
+private val INITIAL_MODULE_3_Drive_TRANSLATION =
+    getTranslation3d(
+        x = Meters.of(-0.25448),
+        y = Meters.of(-0.25448),
+        z = Meters.of(0.0508)
+    )
```

Comment:
You can make a list based on the distances of the modules from the center of rotation.
Also, if you need to specify all three axes, just do `Translation3d(1,2,3)`.